test/controllers/PenaltyDetailsController.test.ts

```diff
@@ -0,0 +1,48 @@
+import 'reflect-metadata'
+import { createApplication } from '../ApplicationFactory';
+import * as request from 'supertest'
+import { createSubstituteOf } from '../SubstituteFactory';
+
+import '../../src/controllers/PenaltyDetailsController';
+import { RedisService } from '../../src/services/RedisService';
+import { BAD_REQUEST, OK } from 'http-status-codes';
+import { PenaltyReferenceDetails } from '../../src/models/PenaltyReferenceDetails';
+
+const app = createApplication(container => {
+    container
+        .bind(RedisService).toConstantValue(createSubstituteOf<RedisService>(service => {
+            service.ping().returns(true)
+        }));
+});
+
+describe('PenaltyDetailsController', () => {
+
+    it('should return 200 when trying to access the penalty-reference page', async () => {
```

Comment:
May I suggest you follow example set by 'test/controllers/OtherReasonController.test.ts', specifically:
- grouping GET and POST requests
- content assertions such as:
  - checking page heading
  - checking error summary heading
  - checking error messages